
GL_UNSIGNED_BYTE does produce incorrect (incorrectly byte-swapped) results
with both software rendering (MESA) and the radeon accelerator ...

  - Rene Rebe <rene@rocklinux.org>

--- evas-2004-02-25/src/lib/engines/gl_common/evas_gl_private.h	2004-02-04 05:38:34.000000000 +0100
+++ evas-2004-02-25-hacking/src/lib/engines/gl_common/evas_gl_private.h	2004-02-27 05:05:52.000000000 +0100
@@ -78,8 +78,10 @@
 /* evas ARGB pixel config */
 #define NATIVE_PIX_FORMAT GL_BGRA
 /* fast on vidia */
-/*#define NATIVE_PIX_UNIT   GL_UNSIGNED_INT_8_8_8_8_REV*/
+/* #define NATIVE_PIX_UNIT   GL_UNSIGNED_INT_8_8_8_8_REV /*
 /* fast on ati compared to GL_UNSIGNED_INT_8_8_8_8_REV */
-#define NATIVE_PIX_UNIT   GL_UNSIGNED_BYTE
+/* #define NATIVE_PIX_UNIT   GL_UNSIGNED_BYTE */
+/* works compared to GL_UNSIGNED_BYTE -ReneR */
+#define NATIVE_PIX_UNIT   GL_UNSIGNED_INT_8_8_8_8_REV
 
 #endif
